### PR TITLE
Add 2a scenario with HA, sbd and kvm

### PR DESCRIPTION
**Pull request by vmoravec**
Contains yaml config for crowbar batch.
More on this at https://trello.com/c/Y4pfBThl/680-3-s32p3-qa-infra-mkcloud-ha-scenario-with-kvm-config-2a

### CODE REVIEW

File: scripts/scenarios/cloud6/qa-scenario-2a-sbd-kvm.yaml [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+      cinder-controller:
+      - cluster:services
+      cinder-volume:
+      - "@@data1@@"
+      - "@@data2@@"
+
+- barclamp: neutron
+  attributes:
+    rabbitmq_instance: default
+    keystone_instance: default
```
Comment (jsuchome) [inline scripts/scenarios/cloud6/qa-scenario-2a-sbd-kvm.yaml]: I guess this is not needed

File: scripts/scenarios/cloud6/qa-scenario-2a-sbd-kvm.yaml [hunk 2]
```diff
@@ -0,0 +1,288 @@
...
+    ha:
+      storage:
+        shared:
+          device: 10.162.26.129:/var/qa2/ha-database
+          fstype: nfs
+          options: nfsvers=3
+  deployment:
+    elements:
+      database-server:
+      - cluster:services
```
Comment (jsuchome) [inline scripts/scenarios/cloud6/qa-scenario-2a-sbd-kvm.yaml]: Shouldn't this (and rabbitmq) be cluster:data?

File: scripts/scenarios/cloud6/qa-scenario-2a-sbd-kvm.yaml [hunk 3]
```diff
@@ -0,0 +1,288 @@
...
+    api:
+      protocol: https
+    ssl:
+      generate_certs: true
+      insecure: true
+    default_store: swift
+  deployment:
+    elements:
+      glance-server:
+      - cluster:data
```
Comment (jsuchome) [inline scripts/scenarios/cloud6/qa-scenario-2a-sbd-kvm.yaml]: Really cluster:data here? I'd expect cluster:services...

File: scripts/scenarios/cloud6/qa-scenario-2a-sbd-kvm.yaml (hunk 2 above)
Comment (vmoravec) [inline scripts/scenarios/cloud6/qa-scenario-2a-sbd-kvm.yaml]: I'm not sure, but the storage for database is on nfs, and the postgresql-server is usually running on a controller, that's why I put it on the services cluster.

File: scripts/scenarios/cloud6/qa-scenario-2a-sbd-kvm.yaml (hunk 3 above)
Comment (vmoravec) [inline scripts/scenarios/cloud6/qa-scenario-2a-sbd-kvm.yaml]: Similar case as the postgresq server, here I agree it should rather be cluster:services.

File: scripts/scenarios/cloud6/qa-scenario-2a-sbd-kvm.yaml (hunk 2 above)
Comment (vuntz) [inline scripts/scenarios/cloud6/qa-scenario-2a-sbd-kvm.yaml]: Generally, the data cluster is for database and rabbitmq.

**Comment by mjura** [comment]
For me it looks ok, +1

**Comment by jsuchome** [comment]
+1

**Pull request merged**